### ISSUE TITLE
Paste into TextField widget not showing all content

### ISSUE THREAD

**Issue by stevey6404** [FlutterFlow/flutterflow-issues#2229]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I'm facing an issue with paste into TextField and I'm looking for a solution.

When I do a 'share’ from Yelp iOS app and copy action, and subsequently paste it into the text input field, it stops after the first line and doesn’t paste the URL. This applies to both iOS web and native app. 

Eg.
Check out Tradewinds Restaurant
https://yelp.to/VBamzDE5Jj
Becomes -> Check out Tradewinds Restaurant

Video: https://drive.google.com/file/d/12lwSKevQZjOU-fBvkkgAPd11eK7Bq1rl/view

### Expected Behavior

The full text should be pasted

Eg.
Check out Tradewinds Restaurant
https://yelp.to/VBamzDE5Jj
should remain:
Check out Tradewinds Restaurant
https://yelp.to/VBamzDE5Jj

### Steps to Reproduce

1. Create new page
2. Create text input field
3. Share from Yelp iOS App > select copy
4. Paste into text input field 

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

ITESz/Hqx89grs9d0bqNdO1AhQMjNlx7bZAnktV+WRMhfb2sOqQue/T8bhNLYtSodGNYOGGL8VsH+NaJi/OSE/Q7Gy+YbKJz+q5XQRfiITuieJvRPJeafUFSRNFhCUjC1MORnx1BOtB1LXAT70zyevCNG3qCf9qOYwx5e6fDbOY=

### Context

Copy and pasting applies to a lot of apps. I'm trying to obtain full text so we can use an API call to query the link. 

### Visual documentation

Video: https://drive.google.com/file/d/12lwSKevQZjOU-fBvkkgAPd11eK7Bq1rl/view

### Additional Info

Raised this with FF Support and they concluded it was a bug and was informed to submit a bug report. 

### Environment

```markdown
- FlutterFlow version: 4.0
- Platform: iOS and mobile web
- Browser name and version: Chrome Version 120.0.6099.216
- Operating system and version affected: iOS mobile and iOS app
```

**Comment by stevey6404** [comment]
waiting for respnose...

**Comment by ldm616** [comment]
waiting for response...

**Comment by stevey6404** [comment]
bump..

**Comment by paulperez-dev** [comment]
Hi @stevey6404, I'm sorry for the delay in the response, we are trying to get up to speed with everything. Are you still facing the issue? Did you look into the `Max Lines` property?

**Comment by stevey6404** [comment]
Yes, no line is set for Max Lines. Even if we set max lines to 20. The URL is not pasted.

**Comment by paulperez-dev** [comment]
@stevey6404 I'm not able to find the element reported in your project. Can you please share a new bug report code? It would be better if it is from a blank project.<br><br>Is this happening only when you share from that app or any app?<br>If you paste the text in another app, like notes, is the URL pasted?

**Comment by paulperez-dev** [comment]
Hi @stevey6404, thanks for the evidence and all the info! I will send this to the ENG team so they can work on this. Thanks for your feedback!

**Comment by ldm616** [comment]
@paulperez-dev - What is the status now? I see it was added as a confirmed bug and assigned to @takashiff and then it looks like the bug label and assignment were both removed?
